### PR TITLE
Comments the root directory.

### LINKED ISSUE #127: Comment root package

### PR DESCRIPTION
**Pull request by JuliusAlexanderIV**
closes #127

### CODE REVIEW

File: app/src/main/java/xyz/whereuat/whereuat/LoginActivity.java [hunk 1]
```diff
@@ -55,42 +62,64 @@ protected void onCreate(Bundle savedInstanceState) {
...
+    /**
+     * This function should be overridden so the receiver can be unregistered when it is not needed.
+     */
     @Override
     protected void onPause() {
         try {
             LocalBroadcastManager.getInstance(this).unregisterReceiver(mTokenReceiver);
-        } catch (IllegalArgumentException e) {}
+        } catch (IllegalArgumentException e) {
+            Log.d(TAG, "Unable to unregister the TokenReceiver");
```
Comment (JuliusAlexanderIV) [inline app/src/main/java/xyz/whereuat/whereuat/LoginActivity.java]: I just sucked it up and added these logs so I could get rid of warnings.

File: app/src/main/java/xyz/whereuat/whereuat/LoginActivity.java [hunk 2]
```diff
@@ -109,17 +138,29 @@ public void onErrorResponse(VolleyError error) {
         }
     }
 
+    /**
+     * This function shows the text input for entering the verification code and hides the text
+     * input for entering the user's phone number .
```
Comment (kangp3) [inline app/src/main/java/xyz/whereuat/whereuat/LoginActivity.java]: What happened to the period here

File: app/src/main/java/xyz/whereuat/whereuat/LoginActivity.java [hunk 3]
```diff
@@ -109,17 +138,29 @@ public void onErrorResponse(VolleyError error) {
...
     public void createNewAccount(View v) {
         startService(new Intent(this, RegistrationIntentService.class));
     }
 
-    /*
-     * Checks that the phone number is in the valid form.
+    /**
+     * Checks that the phone number is in a valid form.
+     *
+     * @param phone_number the hone number to check for validity
```
Comment (kangp3) [inline app/src/main/java/xyz/whereuat/whereuat/LoginActivity.java]: s/hone/phone

File: app/src/main/java/xyz/whereuat/whereuat/MainActivity.java [hunk 4]
```diff
@@ -87,7 +94,14 @@ protected void onCreate(Bundle savedInstanceState) {
         this.startService(intent);
     }
 
-    //
+    /**
+     * This is called when the loader is first made. It is necessary to implements the LoaderManager
```
Comment (kangp3) [inline app/src/main/java/xyz/whereuat/whereuat/MainActivity.java]: s/implements/implement

File: app/src/main/java/xyz/whereuat/whereuat/MainActivity.java [hunk 5]
```diff
@@ -87,7 +94,14 @@ protected void onCreate(Bundle savedInstanceState) {
...
+     * callbacks.
+     *
+     * @param id the id of the loader, which is not needed because there is only one
+     * @param args miscellaneous arguments to help make the loader, of which there are none
```
Comment (kangp3) [inline app/src/main/java/xyz/whereuat/whereuat/MainActivity.java]: These params are next level

File: app/src/main/java/xyz/whereuat/whereuat/MainActivity.java [hunk 6]
```diff
@@ -250,26 +302,29 @@ public void onActivityResult(int req_code, int result_code, Intent data) {
         if (name != null && phone != null) {
             String[] cols = {};
             QueryCommand query = ContactUtils.buildSelectContactByPhoneCommand(this, phone, cols);
+            // Execute the command that checks if the contact is already in the database.
             new DbTask() {
                 @Override
                 public void onPostExecute(Object result) {
                     Cursor c = (Cursor) result;
                     boolean exists = c.getCount() > 0;
+                    // Make sure the contact does not exist before inserting them into the database.
```
Comment (kangp3) [inline app/src/main/java/xyz/whereuat/whereuat/MainActivity.java]: s/them/it
I'm objectifying our contacts.

File: app/src/main/java/xyz/whereuat/whereuat/MainActivity.java (hunk 5 above)
Comment (JuliusAlexanderIV) [inline app/src/main/java/xyz/whereuat/whereuat/MainActivity.java]: Not totally sure what they're for lol. Just an overridden function that needs them

**Pull request merged**